pkg/miniogw/config.go

```diff
@@ -143,11 +149,11 @@ func (c Config) GetBucketStore(ctx context.Context, identity *provider.FullIdent
 	segments := segment.NewSegmentStore(oc, ec, pdb, rs, c.MaxInlineSize)
 
 	// segment size 64MB
-	stream, err := streams.NewStreamStore(segments, c.SegmentSize)
+	stream, err := streams.NewStreamStore(segments, c.SegmentSize, c.EncryptionKey, c.EncryptionBlockSize)
```

Comment:
we probably want to complain if the EncryptionKey is empty, and we probably want to default EncryptionBlockSize to something, and then default ErasureShareSize to be a multiple of EncryptionBlockSize if ErasureShareSize is not set.

Reply:
@jtolds I thought that we want the encryption block to be at least one stripe, so we can claim no data can be decrypted without having at least the required (20) number of erasure shares.

So, the EncryptionBlockSize should be a multiple of (ErasureShareSize * MinThreshold) instead of having the ErasureShareSize to be a multiple of EncryptionBlockSize.

Did I get it wrong?